frontend/src/modules/entitydetails/components/EntityDetails.js

```diff
@@ -195,6 +209,7 @@ export class EntityDetailsBase extends React.Component<InternalProps, State> {
                 deleteTranslation={ this.deleteTranslation }
                 updateTranslationStatus={ this.updateTranslationStatus }
                 updateEditorTranslation={ this.updateEditorTranslation }
+                customMachinerySearch={ this.customMachinerySearch }
```

Comment:
This sounds like a variable name and not a function. Maybe rename it to something like `searchMachinery`?